docs/README.md

```diff
@@ -21,4 +26,8 @@ To ask a question or get support, you can join my [Discord Server](https://disco
 [Forums](https://forums.morpheus636.com)
 
 ## Contributing
-To contribute to this project, view the Contribution Docs found in [CONTRIBUTING.md](CONTRIBUTING.md)
\ No newline at end of file
+To contribute to this project, view the Contribution Docs found in [CONTRIBUTING.md](CONTRIBUTING.md)
+
+---
+
```

Comment:
Put in a credits subhead. Probably a H2 or H3.

Reply:
Oh sorry I meant above where the actual credits are, put a subhead that says
## Credits